.changelog/4619777.md

```diff
@@ -0,0 +1,9 @@
+---
+applies_to: ["server"]
+authors: ["drganjoo"]
+references: ["smithy-rs#3803"]
+breaking: false
+new_feature: true
+bug_fix: false
+---
+Setting the `addValidationExceptionToConstrainedOperations` codegen flag adds `aws.smithy.framework#ValidationException` to operations with constrained inputs that do not already have this exception added.
```

Comment:
I'd add the example you have in your PR description here so users can see how to set the flag.